### ISSUE TITLE
[Bug?] Primary instance already running; but PID of this instance is not active

### ISSUE THREAD

**Issue by bsdooby** [dail8859/NotepadNext#649]
### Operating System and Version

macOS Sequoia

### Distribution

dmg

### Description

I cannot start `NotepadNext`, as there seems to be an already running instance.
No process exists. Are there any stale lock files stored?

### Steps to Reproduce

1. Go into the App bundle: `NotepadNext.app/../MacOS/`
2. Start `./NotepadNext`
3. Get log message: `=== ... [     1.250] I: Primary instance already running. PID: 45127
[     1.357] I: Secondary instance closing...`
4. PID shown does not exist (no instance of `NotepadNext` is actually running)
5. Thus, `NotepadNext` cannot be started (anymore) 

### Additional Details

[NotepadNext.log](https://github.com/user-attachments/files/17422949/NotepadNext.log)

**Comment by dail8859** [comment]
According to your attached log file:

```
[     1.250] I: Notepad Next v0.7
[     1.250] I: Build Date/Time: Apr  5 2024 16:36:24
[     1.250] I: Qt: 6.7.0
```

This must have been a custom or test build. There is a known issue (https://github.com/itay-grudev/SingleApplication/issues/190#issuecomment-1950231485) with multi-instance support for anything past Qt 6.5, thus Qt 6.7 will have this kind of error

**Comment by bsdooby** [comment]
I did not do any custom builds; but downloaded it from your GitHub repo. I
am puzzled. What is your proposed solution?

**Comment by dail8859** [comment]
Use the latest v0.8 version from this repository and it should work fine.

**Comment by bsdooby** [comment]
I did use the latest version: I observe the following: NotepadNext built w/
Q 6.6 does yield the error message, the version using Qt 6.5 runs just fine.
That is super strange...

**Comment by dail8859** [comment]
> NotepadNext built w/ Q 6.6 does yield the error message, the version using Qt 6.5 runs just fine

Yes that is expected. This is why all published versions are built with Qt 6.5

**Comment by bsdooby** [comment]
All good then; I always prefer the official versions ;)
